hardhat.config.ts

```diff
@@ -39,9 +39,9 @@ const config: HardhatUserConfig = {
       chainId: 1101,
       tags: ["local"],
     },
-    ropsten: {
+    goerli: {
       url: process.env.CHAIN_API_URL || "",
-      chainId: 3,
+      chainId: 5,
       accounts: process.env.CONTRACT_OWNER_ACCOUNT_PRIVATE_KEY
         ? [process.env.CONTRACT_OWNER_ACCOUNT_PRIVATE_KEY]
```

Comment:
Do we want to make similar change to [this one](https://github.com/keep-network/keep-core/pull/3111) here?